internal/backend/policy/opapolicy/opa.go

```diff
@@ -21,18 +20,17 @@ import (
 	"go.autokitteh.dev/autokitteh/internal/backend/policy"
 )
 
-const defaultConfig = "default"
+// Embedded config path in opa_bundles.FS.
+const embeddedDefaultConfigPath = "default"
 
 type Config struct {
-	// If empty, use bundled config with the name `defaultConfig`.
-	// If begings with "!", use bundled config with that name (without the "!" prefix).
-	ConfigPath string `koanf:"config_path"`
+	ConfigPath         string `koanf:"config_path"`       // if empty, use embedded default config.
```

Comment:
the config path is not the policy path. the config path is a path to server configuration where it can specify in that configuration where the policy is. see https://www.openpolicyagent.org/docs/latest/configuration/.

if an attacker can modify our deployment env vars, we have much, much bigger issues.